Training/1.0 Introduction.md

```diff
@@ -12,37 +12,65 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+# int
+a = 100
+# float
+b = 1.2
+# string
+c = 'acc'
```

Comment:
Good attempt, but quite a few are missing here. Please check python documentation for how to use boolean, dictionaries, tuples and lists.